app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+  skip_before_action :require_login, only: [:new, :create]
+
```

Comment:
:require_login is a standard controller filter in Rails. (It was one of the things Jeremy introduced recently.)